web/static/embed.go

```diff
@@ -45,9 +45,3 @@ var SlackWebContent embed.FS
 
 //go:embed twilio/connect
 var TwilioWebContent embed.FS
-
-//go:embed descopelogin/index.html
-var DescopeIndex string
```

Comment:
`embed.go` is reasonable, because it needs to be at the relative root. What doesn't belong here is go files that control/render dynamic templates, e.g. `github/index.html`.

Reply:
what i did is added some templating logic, i'd rather it stay as now. unless @daabr insists.